In-house patch to remove unnecessary dependencies from Nova's
requirements files. The specific reasons are as follows:

babel			Not applicable

oslo.rootwrap		Not applicable to Solaris

paramiko		Not applicable

--- nova-13.1.0/requirements.txt.~1~	2016-06-14 07:45:49.000000000 -0800
+++ nova-13.1.0/requirements.txt	2016-06-27 18:39:45.611946372 -0800
@@ -22,8 +22,6 @@ PrettyTable<0.8,>=0.7 # BSD
 sqlalchemy-migrate>=0.9.6 # Apache-2.0
 netaddr!=0.7.16,>=0.7.12 # BSD
 netifaces>=0.10.4 # MIT
-paramiko>=1.16.0 # LGPL
-Babel!=2.3.0,!=2.3.1,!=2.3.2,!=2.3.3,>=1.3 # BSD
 iso8601>=0.1.9 # MIT
 jsonschema!=2.5.0,<3.0.0,>=2.0.0 # MIT
 python-cinderclient>=1.3.1 # Apache-2.0
@@ -44,7 +42,6 @@ oslo.reports>=0.6.0 # Apache-2.0
 oslo.serialization>=1.10.0 # Apache-2.0
 oslo.utils>=3.5.0 # Apache-2.0
 oslo.db>=4.1.0 # Apache-2.0
-oslo.rootwrap>=2.0.0 # Apache-2.0
 oslo.messaging>=4.0.0 # Apache-2.0
 oslo.policy>=0.5.0 # Apache-2.0
 oslo.i18n>=2.1.0 # Apache-2.0
